ush/config_defaults.sh

```diff
@@ -1576,6 +1578,7 @@ PPN_PROC_BUFR="1"
 PPN_RUN_REF2TTEN="1"
 PPN_RUN_NONVARCLDANL="1"
 PPN_RUN_GRAPHICS="12"
+PPN_PYTHON_ENSMEAN="18"
```

Comment:
Does using all available processors help speed?

Reply:
I was assuming 36 cores for 36 members ... what is your recommendation for the NNODES_PYTHON_ENSMEAN and PPN_PYTHON_ENSMEAN? Would greatly appreciate your inputs!

Reply:
I don't think I've done anything special here for parallelism, but the script may be able to utilize a full node for all its memory. It definitely will not try to use more than a single node. I'd say 1 node and 40 ppn would probably be best until we can start using dask to speed things up.

Reply:
If I use ppn 40, does it mean I can only run it on nodes with 40 cores or more?

Reply:
Good point! That is true. Maybe this number of tasks is fine, but then use the `--exclusive` flag?

Reply:
Changed it to be 1 node and 24 ppn since most nodes have 24 or more cores. And added  --exclusive flag. Thank you!

Reply:
My understanding is that the --exclusive flag is not recommend anymore.  
"If the job does not share nodes with other users, do not use "--cpus-per-task", and add "--exclusive" to your batch file. The "--exclusive" option means that the process memory limit can be up to the total available node memory. However, when "--cpus-per-task" is there, the "--exclusive" will not work."
[https://rdhpcs-common-docs.rdhpcs.noaa.gov/wiki/index.php/Running_and_Monitoring_Jobs_on_Jet_and_Hera(Theia)_-_SLURM#Setting_the_--[…]task_parameter](https://rdhpcs-common-docs.rdhpcs.noaa.gov/wiki/index.php/Running_and_Monitoring_Jobs_on_Jet_and_Hera(Theia)_-_SLURM#Setting_the_--cpus-per-task_parameter)